File: JavaScript/removeElement.js
/**
 * https://leetcode.com/problems/remove-element/
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 * Time O(n)
 */
var removeElement = function(nums, val) {
    for (var i = 0, len = nums.length; i < len; i++) {
        if (nums[i] === val) {
            nums.splice(i, 1);
            i--;
        }
    }
    return nums.length;
};
console.log(removeElement([1, 1, 1, 2, 3, 3], 1));

File: Python/removeElement.py
def removeElement(nums, val):
    """
    :type nums: List[int]
    :type val: int
    :rtype: int
    """
    for n in nums[:]:
        if n == val:
            nums.remove(n)
    return len(nums)


print removeElement([8], 7)
